code/modules/clothing/shoes/colour.dm

```diff
@@ -96,11 +96,8 @@
 
 /obj/item/clothing/shoes/sneakers/orange/attackby(obj/H, loc, params)
 	..()
-	if ((istype(H, /obj/item/weapon/restraints/handcuffs) && !( src.chained )))
-		//H = null
-		if (src.icon_state != "orange") return
-		if(istype(H, /obj/item/weapon/restraints/handcuffs/cable))
-			return 0
+	// Note: not using istype here because we want to ignore all subtypes
+	if (H.type == /obj/item/weapon/restraints/handcuffs && !chained && icon_state == "orange")

```

Comment:
I'll get rid of the color check since it's an edge case anyway (orange sneekers that have been recolored). But like _I_ said on IRC, we don't want anything but the standard handcuffs to work in this case, so the explicit type check is fine.